turbinia/client.py

```diff
@@ -710,6 +710,65 @@ def format_task_statistics(
     report.append('')
     return '\n'.join(report)
 
+  def format_request_status(self, instance, project, region, days=0):
+    """Formats the recent history for Turbinia Requests.
+
+    Args:
+      instance (string): The Turbinia instance name (by default the same as the
+          INSTANCE_ID in the config).
+      project (string): The name of the project.
+      region (string): The name of the zone to execute in.
+      days (int): The number of days we want history for.
+    Returns:
+      String of Request status
+    """
+    # Set number of days to retrieve data
+    num_days = 7
+    if days != 0:
+      num_days = days
+    task_results = self.get_task_data(instance, project, region, days=num_days)
+    if not task_results:
+      return ''
+
+    # Create dictionary of request_id: saved_paths
+    request_dict = {}
+    for result in task_results:
```

Comment:
It would be nice if we could also print out the last_update time for the request, and order the report below by those times.  That would make this output a lot more useful for debugging.  That would probably require changing the `request_dict` object to track more things though.  Maybe instead of the `request_dict` mapping the `request_id` to just the saved paths, you can map it to a another dict with `saved_paths`, `requester`, `last_update` keys (and this would eliminate the need to loop over the tasks again to get the `requester` down below).   If we're gathering all this other data, another nice to have would be the task count for each request too.

Reply:
Ok so as soon as I grab `task_results` I'm doing a sort on the `last_update` field to grab the latest timestamps and print from there. Added in `last_update`, `task_count` as well to the summary.